.gitignore

```diff
@@ -6,7 +6,7 @@ venv/
 kubevirtci/
 **/RCS/**
 .*.swp
-
+aws/
```

Comment:
?

Reply:
one of the skipper make created this folder - I added it to the ignore so it won't get merged by mistake